lib/unexpectedMitm.js

```diff
@@ -1047,22 +1052,30 @@ module.exports = {
                                  * reject the assertion. This is only safe with
                                  * Unexpected 10.15.x and above.
                                  */
+
+                                exchangeParts.push(e);
                                 reject(e);
                             }
                         });
 
                         function assertMockResponse(mockResponse, mockResponseError) {
                             var mockRequest = new messy.HttpRequest(requestProperties);
+                            var httpPart = {
```

Comment:
Not sure I understand the `httpPart` variable name, what does it mean? :)

Reply:
Yeah, I was using `part` to mean a portion of the http conversation - at minimum to pe consistent with the rest of the file it wants the http prefix taken off. But I'm open for naming suggestions.. :D